ntoskrnl/cc/view.c

```diff
@@ -42,6 +42,17 @@
 
 /* GLOBALS *******************************************************************/
 
+// FIXME: Move elsewhere!
+#if !defined(_MSC_VER) || _MSC_VER >= 1900
+#define RTL_CONSTANT_LARGE_INTEGER(quad_part) {.QuadPart = (LONGLONG)(quad_part)}
```

Comment:
The casts are not needed here. If anything, they belong in he caller (who is doing math)

Reply:
Right... I blame the fever distorting my ability to think. :p